src/NextGen/DonationForm/Routes/DonationFormViewRoute.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Give\NextGen\DonationForm\Routes;
+
+use Give\NextGen\DonationForm\Controllers\DonationFormViewController;
+use Give\NextGen\DonationForm\DataTransferObjects\DonationFormViewRouteData;
+
+/**
+ * @unreleased
+ */
+class DonationFormViewRoute
+{
+    /**
+     * @unreleased
+     *
+     * @return void
```

Comment:
This says it returns void, but it has a return statement. Also, what happens if the view isn't valid? Is it intended to fail silently? If so, let's return early with a comment explaining that.

Reply:
Yes it's intended to fail silently because it's using `template_redirect` so unless the route has the query params we want just move on.  doc updated 09c17aed